hooks/hookstage/bidderrequest.go

```diff
@@ -0,0 +1,19 @@
+package hookstage
```

Comment:
The reason why we renamed the file(s) (the variables as well) was because initially we did not use the correct namings. We plan to use `bidder-request` as the stage code and stage name because 1) we want to follow the PRD namings and 2) there is a difference between bid request and bidder request entities. So in order to avoid any possible misunderstanding or collision in the future we decided to rename all respective parts from our code.

Reply:
Sure, we'll update them as well